docs/rest_api.rst

```diff
@@ -301,7 +302,7 @@ Querying Information About Channels and Tokens
 
    :statuscode 200: Successful query
    :statuscode 302: If the user accesses the channel link endpoint
-   :statuscode 404: If the token does not exist/the token address is invalid
+   :statuscode 404: If the token does not exist/the token address is not valid eip55-encoded
```

Comment:
please consistent about space before and after slash. "exist/the" or "exist / the" as above